src/Apps/Comparables/Filter.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box, Flex, Spacer } from "@artsy/palette"
+import { CollectArtworkGridRefetchContainer as ArtworkFilter } from "Apps/Collect/Components/ArtworkGrid/CollectArtworkFilterRefetch"
+import { PriceRange } from "Apps/Collect/Components/Filters/PriceRange"
+import { FilterState } from "Apps/Collect/FilterState"
+import { AttributionClassFilter } from "Apps/Comparables/Filter/AttributionClassFilter"
+import { MediumFilter } from "Apps/Comparables/Filter/MediumFilter"
+import { ContextConsumer, ContextProps } from "Artsy"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Toggle } from "Styleguide/Components/Toggle"
+import { Subscribe } from "unstated"
+
+export class Comparables extends Component<ContextProps> {
+  renderPriceRange(filters: FilterState, mediator) {
+    const [initialMin, initialMax] = filters.priceRangeToTuple()
+
+    return (
+      <PriceRange
+        allowCross={false}
+        min={FilterState.MIN_PRICE}
+        max={FilterState.MAX_PRICE}
+        step={50}
+        defaultValue={[initialMin, initialMax]}
+        onAfterChange={([min, max]) => {
+          const minStr = min === FilterState.MIN_PRICE ? "*" : min
+          const maxStr = max === FilterState.MAX_PRICE ? "*" : max
+
+          filters.setFilter("price_range", `${minStr}-${maxStr}`, mediator)
+        }}
+      />
+    )
+  }
+  render() {
+    return (
+      <ContextConsumer>
+        {({ mediator }) => {
+          return (
+            <Subscribe to={[FilterState]}>
+              {(filters: FilterState) => {
```

Comment:
Can this subscribe call be moved down to line 76 and then passed in as a prop? That way it will be a bit more useful throughout the class.

Reply:
I'll give that a shot.

Reply:
Yea, that did clean up this file.